pages/k8s/1.18/components.md

```diff
@@ -0,0 +1,365 @@
+---
+wrapper_template: kubernetes/docs/base_docs.html
+markdown_includes:
+  nav: kubernetes/docs/shared/_side-navigation.md
+context:
+  title: Components of Charmed Kubernetes 1.18
+  description: Detailed description of Charmed Kubernetes release
+keywords: component, charms, versions, release
+tags:
+    - reference
+sidebar: k8smain-sidebar
+permalink: 1.18/components.html
+layout:
+    - base
+    - ubuntu-com
+toc: false
+bundle_revision: '426'
```

Comment:
This was recently updated b/c of the hacluster interface updated, I think you want this to be 428